htdocs/manage/profile/index.bml

```diff
@@ -693,12 +714,29 @@ body<=
             # set userprops
             my %prop;
             foreach my $uprop (@uprops) {
+                next if $legacy_service_props{$uprop};
                 my $eff_val = $POST{$uprop};   # effective value, since 0 isn't stored
                 $eff_val = "" unless $eff_val;
                 $prop{$uprop} = $eff_val;
             }
             $u->set_prop( \%prop, undef, { skip_db => 1 } );
 
+            # update external services
+            my %services = map { $_->{name} => $_ } @{ DW::External::ProfileServices->list };
+            my %new_accts;
+
+            foreach my $ct (1..26) {
+                my $name = $POST{"extservice_name_$ct"};
+                next unless $name;
+                my $val = $POST{"extservice_val_$ct"} // '';
+                # this isn't great but it's better than not enforcing maxlen at all
```

Comment:
should we should use the `LJ::text_trim` (I think that's the name) so that we don't cut multibyte characters in half? (I think that's a risk here?)

Reply:
I did use that slightly further down, in the save_profile_accts method that gets called right after this, but I guess it would make sense to do it here instead.